src/helpers.rs

```diff
@@ -2,7 +2,7 @@ use std::mem;
 
 use rustc::ty::{self, layout::{self, Size, Align}};
 use rustc::hir::def_id::{DefId, CRATE_DEF_INDEX};
-use rustc::mir;
+use rustc::{mir, err_unsup_format};
```

Comment:
Is this needed? The `use crate::*` should take care of that...

Reply:
It does, I just corrected it